mkosi/__init__.py

```diff
@@ -73,7 +73,7 @@
     run_workspace_command,
     spawn,
 )
-from mkosi.types import PathString
+from mkosi.types import CommandLine, CommandLineReadOnly
```

Comment:
Let's switch this around, CommandLine should be read-only by default and we should add MutableCommandLine for when we want to be able to modify it.